up-transport-vsomeip/src/lib.rs

```diff
@@ -197,6 +197,17 @@ impl UPTransportVsomeip {
         config_path: Option<&Path>,
         runtime_config: Option<RuntimeConfig>,
     ) -> Result<Self, UStatus> {
+        let check_ue_version_major: Result<u8, _> = uri.ue_version_major.try_into();
+        if check_ue_version_major.is_err() {
+            return Err(UStatus::fail_with_code(
+                UCode::INVALID_ARGUMENT,
+                format!(
+                    "uri's ue_version_major doesn't fit allotted 8 bits: uri.ue_version_major: {}",
+                    uri.ue_version_major
+                ),
+            ));
+        }
+
```

Comment:
FMPOV this check should be in up-rust. I guess that is what https://github.com/eclipse-uprotocol/up-rust/issues/209 is about, right?